<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addCity">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>增加城市</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;请选择您的操作&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteCity">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>删除城市</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeCity">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>修改城市信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addFlights">
    <property name="geometry">
     <rect>
      <x>299</x>
      <y>130</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>增加航班</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteFlight">
    <property name="geometry">
     <rect>
      <x>299</x>
      <y>200</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>删除航班</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeFlight">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>修改航班信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addTrainTrip">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>130</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>增加列车</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteTrainTrip">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>200</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>删除列车</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeTrainTrip">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>270</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>修改列车信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="manageAccount">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>管理用户账号信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>340</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findInfo">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>340</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>101</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>查询航班和列车信息</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
